cni/pkg/nodeagent/server.go

```diff
@@ -93,7 +95,9 @@ func NewServer(ctx context.Context, ready *atomic.Value, pluginSocket string, ar
 	}
 
 	// Create hostprobe rules now, in the host netns
-	hostIptables.DeleteHostRules()
+	if !args.Reconcile {
```

Comment:
Hostprobe rules are not traffic-critical, and for now I would just leave this as it was/not bother doing reconcile logic at all for this.

We really only care about reconciling in-pod rules, and not the (very minimal, noncritical) host-level rules.

Reply:
Understood.
I’ll revert this portion and I will have to update NewIptablesConfigurator to have independent configurations for host and pod. I’ll then set Reconcile for the pod configuration only if ReconcilePodRulesOnStartup == true, while setting Reconcile for the host configuration to always be false.
Currently, they share the same iptables cfg, which results in both having Reconcile set/unset.

Reply:
Done